package.json

```diff
@@ -1,27 +1,32 @@
 {
   "name": "ng-feedhenry",
   "version": "0.2.53",
-  "main": "./src/ngFH.js",
+  "main": "./dist/ngFH.js",
   "description": "AngularJS friendly FeedHenry SDK wrapper.",
   "repository": "feedhenry-staff/ngFeedHenry",
   "scripts": {
-    "test": "make test"
+    "clean": "rm -r dist test/bundle.js",
+    "build": "npm run lint && ./node_modules/.bin/browserify -e ./src/ngFH.js -o ./dist/ngFH.js",
+    "debugbuild": "./node_modules/.bin/browserify -e ./src/ngFH.js -o ./test/bundle.js",
+    "prepublish": "npm run build",
+    "lint": "./node_modules/.bin/eslint .",
```

Comment:
We can actually remove these `node_modules` prefixes, so `./node_modules/.bin/eslint .` can be `eslint .`